packages/truffle-compile/compilerSupplier/loadingStrategies/VersionRange.js

```diff
@@ -90,18 +90,18 @@ class VersionRange extends LoadingStrategy {
     const solcFileName = this.getCachedSolcFileName(commit);
     if (solcFileName) return this.getCachedSolcByFileName(solcFileName);
 
-    const allVersions = await this.getSolcVersions(this.config.versionsUrl);
+    const allVersions = await this.getSolcVersions(0);
```

Comment:
I think we should make the `0` be the default value for the argument, so we don't need to change the interface for that method.